src/pages/SocialGraph.vue

```diff
@@ -7,12 +7,22 @@
         <option value="creator">Creator</option>
         <option value="collector">Collector</option>
       </select>
-      Of
+      of
       <input
         v-model="inputString"
         type="text"
         size="40"
       >
+      , order by
+      <select v-model="orderBy">
+        <option value="price">Price</option>
+        <option value="count">Count</option>
+      </select>
+      , price by
+      <select v-model="priceBy">
+        <option value="class">Class</option>
+        <option value="nft">NFT</option>
+      </select>
```

Comment:
Technically it is class vs NFT, but I think the layman term would be something like `current value`(class) and `purchase price`(nft)?